### ISSUE TITLE
buffer generated by counsel-ag-occur is grep-mode so wgrep fails

### ISSUE THREAD

**Issue by cpbotha** [abo-abo/swiper#1203]
When I do a search with counsel-grep and then activate counsel-occur with `C-c o`, my results look something like this:

```
-*- mode:grep; default-directory: "~/Dropbox/work/code/vtktudoss/" -*-

12 candidates:
./CompVis/vtkCVImageMapToColors.h:13:3:  vtkTypeRevisionMacro(vtkCVImageMapToColors,vtkImageMapToColors);
./CompVis/vtkFixedPointVolumeRayCastCVHelper.h:60:3:  vtkTypeMacro(vtkFixedPointVolumeRayCastCVHelper,vtkFixedPointVolumeRayCastHelper);
./CompVis/vtkCVFixedPointVolumeRayCastMapper.h:50:3:  vtkTypeMacro(vtkCVFixedPointVolumeRayCastMapper,vtkFixedPointVolumeRayCastMapper);
```

When I now activate wgrep mode with `C-x C-q`, edit the results, and then `C-x C-s` no changes are applied.

However, when I do the same search using plain grep, the results look like this:

```
-*- mode: grep; default-directory: "~/Dropbox/work/code/vtktudoss/" -*-
Grep started at Tue Sep 19 17:54:16

./CompVis/vtkCVFixedPointVolumeRayCastMapper.h:50:  vtkTypeMacro(vtkCVFixedPointVolumeRayCastMapper,vtkFixedPointVolumeRayCastMapper);
./CompVis/vtkCVImageMapToColors.h:13:  vtkTypeMacro(vtkCVImageMapToColors,vtkImageMapToColors);
./CompVis/vtkFixedPointVolumeRayCastCVHelper.h:60:  vtkTypeMacro(vtkFixedPointVolumeRayCastCVHelper,vtkFixedPointVolumeRayCastHelper);
```

Note that the column number is not shown. Now wgrep works perfectly, applying all changes.

It looks like plain wgrep is confused by that extra column number. I have the latest wgrep-ag installed, but its setup only runs as part of the ag-mode-hook. I have package ag.el installed, but can't see any ag-mode defined anywhere.

In short: How can I get wgrep working together with counsel-ag?

macos 10.12.6, emacs 25.3.2, counsel 2017091, wgrep 20141016, wgrep-ag 20160923

**Comment by cpbotha** [comment]
I see the poblem in my configuration!

As per https://github.com/abo-abo/swiper/pull/303 I had `counsel-ag-base-command` set to `ag --vimgrep %s` for my setup to work on Windows also.

However, removing this customization (so it reverts back to `ag --nocolor --nogroup %s` fixed by counsel-ag-occur issue!

(I hope this helps someone else. I'll close the issue.)
